hypha/apply/users/templatetags/users_tags.py

```diff
@@ -25,3 +26,12 @@ def can_use_oauth(context):
     user = context.get('user')
 
     return can_use_oauth_check(user)
+
+
+@register.simple_tag(takes_context=True)
+def user_2fa_enabled(context):
```

Comment:
Instead of using context and then getting the user, can we just pass the user as a variable? I think it is already available in the template?

Reply:
Yes, user is there we can use that directly.